sphinx/cmdline.py

```diff
@@ -191,23 +194,62 @@ def main(argv):
 
     # get paths (first and second positional argument)
     try:
-        srcdir = abspath(args[0])
-        confdir = abspath(opts.confdir or srcdir)
-        if opts.noconfig:
-            confdir = None
+        # Check src directory (first positional argument)
+        srcdir = path.abspath(args[0])
```

Comment:
Please use naked `abspath()` instead.
It equals `sphinx.util.ospath.abspath()`, not `os.path.abspath()`.